src/hashtag.js

```diff
@@ -8,7 +8,9 @@ function flowdockHashtag(tokens, idx) {
 }
 
 export default function(md, options) {
-  const hashtag = parser(md, 'hashtag', /#|＃/, FlowdockText.regexen.autoLinkHashtags);
+  const split = "#|＃";
+  const regex = new RegExp("(?:^|$|[^" + tagEnd.slice(1) + ")((?:" + split + ")(?:" + tagCharacter + ")*(?:" + tagEnd + ")+)", "g");
```

Comment:
To reduce future work when figuring out why the `tagEnd.slice(1)` is done, it would be probably good to add a comment about removing the opening brace for the char class.